Formula/libdrm.rb

```diff
@@ -4,22 +4,25 @@ class Libdrm < Formula
   url "https://dri.freedesktop.org/libdrm/libdrm-2.4.104.tar.xz"
   sha256 "d66ad8b5c2441015ac1333e40137bb803c3bde3612ff040286fcc12158ea1bcb"
   license "MIT"
+  revision 1
 
   livecheck do
     url "https://dri.freedesktop.org/libdrm/"
     regex(/href=.*?libdrm[._-]v?(\d+(?:\.\d+)+)\.t/i)
   end
 
-  depends_on "docbook" => :build
-  depends_on "docbook-xsl" => :build
-  depends_on "libxslt" => :build
+  depends_on "docutils" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "libpciaccess"
   depends_on :linux
 
   def install
+    inreplace "meson.build",
+      "find_program('rst2man'",
+      "find_program('#{Formula["docutils"].opt_bin}/rst2man'"
```

Comment:
Is `meson` unable to find `rst2man` without this? It should, because `rst2man` should be in `PATH` during the build.

Reply:
I am not familiar with `meson`, but in my case `find_program` returns the path to the system rst2man from `/usr/bin/rst2man`, which is outdated and does not work correctly.

Reply:
On our CI `/usr/bin/rst2man` does not exist so this should not be an issue there.

It will be only an issue if you build from source.

The docutils formula provides `bin/rst2man.py` which is in the PATH. I guess we should have symlinks from `rst2man` to `rst2man.py` to make this work?

Reply:
Probably you are right, I just noticed that the brewed `rst2man` is created with the `.py` extension. But now I am confused how the path replacement in `find_program` works, because the replaced path is there without the `.py` extension :)

Reply:
https://github.com/freedesktop/mesa-drm/blob/libdrm-2.4.104/meson.build#L264-L265
Now I understood it. Since the path to `find_program` was specified without the `.py` extension, the check for the existence of the `#{Formula["docutils"].opt_bin}/rst2man` executable file was triggered, and in its absence, manuals were just not created. However, in the first case, when the system `rst2man` was found, its execution ended with an error, so the build was failed.

Therefore, in any case, we need to either create a symlink of `rst2man` without the `.py` extension, or change the path in `find_program` to add the `.py` extension, which will be found later in the PATH. 

I updated this commit by adding the `.py` extension in `find_program`.

Reply:
This was resolved by a fix in `docutils`.